documentation/docs/aggregation.md

```diff
@@ -0,0 +1,37 @@
+# What is an aggregation formula?
+
+![Group 3](https://user-images.githubusercontent.com/66801011/224264480-a66d91be-fa04-4994-a719-d390d2297ff4.png)
+
+An aggregation formula is an arithmetic formula that decides how to aggregate the result of an arbitrary amount of sub-aggregations and metrics.
+
+Aggregating metric results is a way of approximating higher level qualitative attributes, such as maintainability or reliability. The importance of an aggregation varies depending on the user, and therefore these aggregations need fit the request of the user. Modu helps combat this by letting the user define what sub-aggregations and metrics aggregate into these higher level attributes, and potentially their importance as well. 
```

Comment:
I don't agree with calling maintainability and reliability qualitative.

Reply:
how about we just change it to: "gregating metric results is a way of approximating higher level attributes."?